test/core/transport/chttp2/too_many_pings_test.cc

```diff
@@ -333,14 +333,17 @@ void VerifyChannelDisconnected(grpc_channel* channel,
                                grpc_completion_queue* cq) {
   // Verify channel gets disconnected. Use a ping to make sure that clients
   // tries sending/receiving bytes if the channel is connected.
-  grpc_channel_ping(channel, cq, reinterpret_cast<void*>(2000), nullptr);
-  grpc_event ev = grpc_completion_queue_next(
-      cq, grpc_timeout_seconds_to_deadline(5), nullptr);
-  GPR_ASSERT(ev.type == GRPC_OP_COMPLETE);
-  GPR_ASSERT(ev.tag == reinterpret_cast<void*>(2000));
-  GPR_ASSERT(ev.success == 0);
-  GPR_ASSERT(grpc_channel_check_connectivity_state(channel, 0) !=
-             GRPC_CHANNEL_READY);
+  grpc_event ev;
+  do {
```

Comment:
I think this would defeat the purpose of the test

Reply:
Can you say why?

Reply:
Some of the tests are written with a logic which say that -
We expect a ping to be sent that will cause the "too_many_pings" logic to fire. When a ping causes the transport to be closed, we expect the next ping to fail. If the next ping succeeds instead, then it means that pings did not go out on the wire as expected.

Reply:
backed this part out for now